src/NzbDrone.Core/Housekeeping/Housekeepers/CleanupOrphanedTrackFiles.cs

```diff
@@ -0,0 +1,26 @@
+using NzbDrone.Core.Datastore;
+
+namespace NzbDrone.Core.Housekeeping.Housekeepers
+{
+    public class CleanupOrphanedTrackFiles : IHousekeepingTask
```

Comment:
Would it be worth changing this name to `CleanupNullIdTrackFiles` or something? It seems more obvious than orphaned, although I don't like the `llI` in the name.

Reply:
Merge per discord convo